kafka-rest/src/main/java/io/confluent/kafkarest/DefaultKafkaRestContext.java

```diff
@@ -33,16 +33,16 @@
 import org.apache.kafka.clients.producer.KafkaProducer;
 import org.apache.kafka.clients.producer.Producer;
 import org.apache.kafka.common.serialization.ByteArraySerializer;
-import org.slf4j.Logger;
```

Comment:
please don't change this, slf4j is only the facade, the change needs to be done from changing the logging library instead (i.e reload4j -> log4j2)

Reply:
@trnguyencflt I though we needed to remove slf4j-reload4j completely. So I was following this guide when making changes - https://logging.apache.org/log4j/2.x/migrate-from-slf4j.html#migrating

Reply:
I think this may be more of what we want - https://logging.apache.org/log4j/2.x/manual/installation.html#impl-core-bridge-slf4j